docs/docs/handling-events.md

```diff
@@ -2,7 +2,115 @@
 id: handling-events
 title: Handling Events
 permalink: docs/handling-events.html
-prev: state-and-lifecycle.html
 ---
 
-TODO
+Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+
+* React events are named using camelCase, rather than lowercase
+* With JSX you pass a function as the event handler, rather than a string
+
+For example, the HTML:
+
+```html
+<button onclick="activateLasers()">Activate Lasers</button>
```

Comment:
I'm not confident that comparisons to HTML are necessary at all.

People who know how HTML event handlers work in 2016 also probably have learned that those are "very bad".

I think maybe we should reframe this section as if the person has never heard of HTML at all.